src/System.Management.Automation/utils/EncodingUtils.cs

```diff
@@ -115,6 +115,11 @@ internal sealed class ArgumentToEncodingTransformationAttribute : ArgumentTransf
     {
         public override object Transform(EngineIntrinsics engineIntrinsics, object inputData)
         {
+            if (inputData is PSObject inputPSObject)
+            {
+                inputData = inputPSObject.BaseObject;
+            }
```

Comment:
```suggestion
            inputData = PSObject.Base(inputData);
```

Reply:
Is there any benefit to this approach? I personally find the if statement clearer in its intent.

Reply:
This is a common pattern. This will work even if there are multiple levels of wrappers in PSObject.

Reply:
I still think the intent is clearer with the if statement but `PSObject.Base` does a few extra things that while won't affect the code here does seem to be a bit more correct.